main.py

```diff
@@ -14,6 +14,7 @@
 from sqlalchemy import create_engine
 from sqlalchemy.orm import sessionmaker
 from models import Account
+from argparse import ArgumentParser
```

Comment:
Super nit-picky, but depending on how fussy one is about imports they might prefer them to be:

```py
from argparse import ArgumentParser
from math import floor
import os
import random
import uuid

from sqlalchemy import create_engine
from sqlalchemy.orm import sessionmaker
from sqlalchemy_cockroachdb import run_transaction

from models import Account
```